benchmarks/asv.conf.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": 1,
+    "project": "Outlines",
+    "project_url": "https://outlines-dev.github.io/outlines/",
+    "repo": "..",
+    "branches": [
+	"HEAD"
+    ],
+    "build_command": [
+        "pip install .[test]",
```

Comment:
This should be in `install_command` instead, no?  I tried it locally with `"install_command": ["in-dir={env_dir} python -mpip install '{wheel_file}[test]'"]`, but it fails with no useful output.  

We need to figure this out before going forward.

Reply:
I ran into issues using a canonical `install_command` / `build_command`, and instead opted to follow this workaround: https://github.com/airspeed-velocity/asv/issues/903#issuecomment-1276110657

Reply:
OK, then we should change it to something like 
```
    "build_command": [
        "in-dir={env_dir} python -mpip install .[test]",
        "python setup.py build",
        "PIP_NO_BUILD_ISOLATION=false python -mpip wheel --no-deps --no-index -w {build_cache_dir} {build_dir}",
    ],
```
which reflects the current build command defaults.